### PR TITLE
Port to new libadwaita specialized rows

### PR DESCRIPTION
**Pull request by oscfdezdz**

### CODE REVIEW

**Comment by flxzt** [comment]
thanks!

The only thing I noticed is that you didn't update the widget variable names in the closures for the `connect_*_notify`/`connect_changed` callbacks. Otherwise it looks good to me.

Porting the custom widget `RnUnitEntry` to a style similar to spin rows should be looked into as well, but it doesn't need to be in this PR.

**Comment by oscfdezdz** [comment]
> The only thing I noticed is that you didn't update the widget variable names in the closures for the `connect_*_notify`/`connect_changed` callbacks. Otherwise it looks good to me.
> 
> Porting the custom widget `RnUnitEntry` to a style similar to spin rows should be looked into as well, but it doesn't need to be in this PR.

They are now addressed!

File: crates/rnote-ui/data/ui/unitentry.ui [hunk 1]
```diff
@@ -23,7 +20,9 @@
     </child>
     <child>
       <object class="GtkDropDown" id="unit_dropdown">
+        <property name="margin-start">12</property>
```
Comment (flxzt) [inline crates/rnote-ui/data/ui/unitentry.ui:23]: A margin of 6 should be enough here

**Comment by flxzt** [comment]
looks good! I like that it brings a more consistent look between different types of rows.

**Pull request merged**